src/main/app/src/app/zaken/zoek/locatie-zoek/locatie-zoek.component.ts

```diff
@@ -225,6 +244,6 @@ export class LocatieZoekComponent implements OnInit, AfterViewInit, OnDestroy {
     }
 
     resultDisplay = (result: any): string => {
```

Comment:
deze methode retoyrneert dan toch string | null ? (of zo, ik kan het mis hebben)